pkg/sql/logictest/testdata/logic_test/row_level_ttl

```diff
@@ -121,6 +120,34 @@ WHERE label LIKE 'row-level-ttl-%'
 ----
 0
 
+# Check the same thing with SET (ttl = off)
+statement ok
+DROP TABLE tbl;
+CREATE TABLE tbl (
```

Comment:
if you make DROP/CREATE two separate statements then you'll save me another edit on https://github.com/cockroachdb/cockroach/pull/76834 :) :) :)

Reply:
you got it!